pymc3/plots.py

```diff
@@ -88,7 +99,7 @@ def histplot_op(ax, data, alpha=.35):
         ax.hist(d, bins=range(mind, maxd + 2, step), alpha=alpha, align='left')
         ax.set_xlim(mind - .5, maxd + .5)
 
-def kdeplot_op(ax, data):
+def kdeplot_op(ax, data, prior=None, prior_alpha=1, prior_style='--'):
```

Comment:
Putting the prior business here seems slightly inelegant to me, other ideas on where it could go? Can we just have a second function?